pygments/lexers/prolog.py

```diff
@@ -226,16 +228,16 @@ class LogtalkLexer(RegexLexer):
             (r'\^', Operator),
             # Strings
             (r'"(\\\\|\\"|[^"])*"', String),
-            # Ponctuation
+            # Punctuation
             (r'[()\[\],.|]', Text),
             # Atoms
-            (r"[a-z]\w*", Text),
-            (r"'", String, 'quoted_atom'),
+            (r"[a-z][a-zA-Z0-9_]*", Text),
```

Comment:
` \w` should be equivalent? Why this change?

Reply:
Is my understanding that `\w` can include Unicode code points besides ASCII characters, which may result in an invalid atom.

Reply:
Ah yes, correct. I'm not sure we do unicode yet, but I guess if the language is limited to ASCII, doesn't hurt to make this explicit. Thanks!

Reply:
Well, Unicode is explicitly selected by re.UNICODE being in the flags for the lexer. If that's not accurate it can just be deleted.

Reply:
That explicit selection is done only in the Prolog lexer, not in the Logtalk lexer. Even there, is not clear. The ISO Prolog standards are mute on Unicode. Some Prolog systems support it, most don't. Among the Prolog systems that do support Unicode, there's no common de facto standard, particularly regarding using of non ASCII code points in identifiers.